cms/static/cms/js/modules/cms.structureboard.js

```diff
@@ -8,7 +8,8 @@ import Plugin from './cms.plugins';
 import { getPlaceholderIds } from './cms.toolbar';
 import Clipboard from './cms.clipboard';
 import URI from 'urijs';
-import DiffDOM from 'diff-dom';
+import { DiffDOM } from 'diff-dom';
+import Promise from 'promise-polyfill';
```

Comment:
Frankly, I am uncertain if polyfills should go into the `webpack.config.js` or into source files. What's the common practice?

Reply:
I don't like it either. Gulp lint gave me:

```/Users/vinitkumar/projects/python/django-cms/cms/tests/frontend/unit/cms.structureboard.test.js
  251:64  error  Promise.resolve() is not supported in IE Mobile 10, IE 10, bb 7  compat/compat
  344:54  error  Promise.resolve() is not supported in IE Mobile 10, IE 10, bb 7  compat/compat
  345:54  error  Promise.resolve() is not supported in IE Mobile 10, IE 10, bb 7  compat/compat
  421:62  error  Promise.resolve() is not supported in IE Mobile 10, IE 10, bb 7  compat/compat
  422:64  error  Promise.resolve() is not supported in IE Mobile 10, IE 10, bb 7  compat/compat
```
So either we don't care about these browsers, in that case, we just change the eslint config and get the lint to pass.

Reply:
IE 10 is 10 years old and, according to [caniuse](https://caniuse.com/?search=promise), has a market share of 0.07%. 

@marksweb: I think we should drop those. Probably this requirement has hold us back from updating some of the packages.

Reply:
@fsbraun @marksweb In that case, I drop these and remove all browsers that don't support Promises and guarantee only modern ones will work.

Reply:
Yeah drop IE support. People should have moved on to edge by now